src/app/playground/actions/updateDevUserRole.ts

```diff
@@ -0,0 +1,25 @@
+"use server";
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { UpdateDevUserRole } from "../components/UpdateDevUserForm";
+import { formatPhoneNumber } from "@/utils/formatPhoneNumber";
+
+export const updateDevUserRole = async (values: UpdateDevUserRole) => {
+  const supabase = await getSupabaseClient();
+  const userPhoneNumber = formatPhoneNumber(values.phoneNumber);
+
+  const { data: updatedDevUser, error } = await supabase
```

Comment:
I believe we can use `updateUser` action here.

Reply:
Sir we can't use `updateUser`  because in this function we are updating on base on `id` , if we use same function, then we need to update `user` on the base on number that is not good approch.